tests/01_03_52-disable_lock_whitelist.js

```diff
@@ -1,121 +1,78 @@
-var btcClient;
-var rskClient;
-var rskClients;
-var pegClient;
-var test;
-var utils;
-let rskTxHelpers;
-
-const NETWORK = bitcoin.networks.testnet;
+const { WHITELIST_CHANGE_PK, WHITELIST_CHANGE_ADDR} = require('../lib/assertions/whitelisting')
 
-const INITIAL_BTC_BALANCE = bitcoin.btcToSatoshis(10);
+let rskTxHelpers;
+let rskTxHelper;
+let btcTxHelper;
+let bridge;
+let MINIMUM_PEGIN_VALUE_IN_BTC;
+let federationAddress;
 
-const WHITELIST_CHANGE_PK = '3890187a3071327cee08467ba1b44ed4c13adb2da0d5ffcc0563c371fa88259c';
-const WHITELIST_CHANGE_ADDR = '87d2a0f33744929da08b65fd62b627ea52b25f8e';
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+  await rskUtils.activateFork(Runners.common.forks.papyrus200);
+};
 
 describe('Disable whitelisting', function() {
-  var addresses;
 
   before(async () => {
-    try{
-      btcClient = bitcoin.getClient(
-        Runners.hosts.bitcoin.rpcHost,
-        Runners.hosts.bitcoin.rpcUser,
-        Runners.hosts.bitcoin.rpcPassword,
-        NETWORK
-      );
-      rskClient = rsk.getClient(Runners.hosts.federate.host);
-      rskClients = Runners.hosts.federates.map(federate => rsk.getClient(federate.host));
-      pegClient = pegUtils.using(btcClient, rskClient);
-      test = pegAssertions.with(btcClient, rskClient, pegClient, rskClients);
-      utils = rskUtilsLegacy.with(btcClient, rskClient, pegClient);
-      rskTxHelpers = getRskTransactionHelpers();
-
-      // Grab the federation address
-      federationAddress = await rskClient.rsk.bridge.methods.getFederationAddress().call();
-      await btcClient.importAddress(federationAddress, 'federations');
-      
-      addresses = await pegClient.generateNewAddress('test');
-      expect(addresses.inRSK).to.be.true;
-      
-      await btcClient.sendToAddress(addresses.btc, INITIAL_BTC_BALANCE);
-      await btcClient.generate(1);
-      await test.assertBitcoinBalance(addresses.btc, INITIAL_BTC_BALANCE, 'Initial BTC balance');
-      
-      var addr = await rskClient.eth.personal.importRawKey(WHITELIST_CHANGE_PK, '');
-      expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
-      
-      await rskClient.eth.personal.unlockAccount(addr, '');
-      await sequentialPromise(10, () => rskUtils.mineAndSync(rskTxHelpers));
-    }
-    catch (err) {
-      throw new CustomError('Lock whitelisting failure', err);
+    rskTxHelpers = getRskTransactionHelpers();
+    rskTxHelper = rskTxHelpers[0];
+    btcTxHelper = getBtcClient();
+
+    const latestActiveForkName = await getLatestActiveForkName();
+    bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+    federationAddress = await bridge.methods.getFederationAddress().call();
+    const minPeginValueInSatoshis = await bridge.methods.getMinimumLockTxValue().call();
+    MINIMUM_PEGIN_VALUE_IN_BTC = Number(satoshisToBtc(minPeginValueInSatoshis));
+
+    if(process.env.RUNNING_SINGLE_TEST_FILE) {
+      await fulfillRequirementsToRunAsSingleTestFile();
```

Comment:
Since you are activating papyrus in this function, this call should probably be above before you try to get the `latestActiveForkName`. Maybe that's the reason why it's not working when trying to run it as a single file

Reply:
didn't help.
all i get as an outcome is

`honza@Jans-MacBook-Pro rootstock-integration-tests % npm run run-single-test-file tests/01_03_52-disable_lock_whitelist.js

> rootstock-integration-tests@1.0.0 run-single-test-file
> node singleTestFileRunner.js tests/01_03_52-disable_lock_whitelist.js

Executing test file 'tests/01_03_52-disable_lock_whitelist.js' in isolation.

> rootstock-integration-tests@1.0.0 test-fail-fast
> mocha -b --timeout 600000



  0 passing (0ms)

Test file 'tests/01_03_52-disable_lock_whitelist.js' ran successfully.
honza@Jans-MacBook-Pro rootstock-integration-tests % 
`